### ISSUE TITLE
Add visual differences for focused/unfocused windows

### ISSUE THREAD

**Issue by Niektory** [widelands/widelands#3852]
Since the behavior of keypresses can change depending on which window has focus, I think you should be able to distinguish focused and unfocused windows visually.
